3rdparty/musl/README.md

```diff
@@ -10,3 +10,15 @@ https://www.musl-libc.org/releases/musl-1.1.19.tar.gz
 Typing **make** installs the source tree under the build directory and
 applies patches to it. Nothing is built though, as building is performed
 from the **libc** directory.
+
+libc-test:
+==========
+
+libc-test is a generic libc testing framework, obtained from this git repo:
+
+```
+git://repo.or.cz/libc-test
```

Comment:
A couple of suggestions:
* Can you clean up the text around the musl libc itself? It's not the right version anymore (OE SDK is on 1.1.21 now) 
* For both the libc and libc-test, can we instead redirect to the [update.make](https://github.com/openenclave/openenclave/blob/master/3rdparty/musl/update.make) file instead as the single source of truth so that we don't need to keep the README.md versioning in sync with the actual update.make? (i.e. put the latest libc-test commit ID in update.make)
* update.make uses the original repo `git://nsz.repo.hu:45100/repo/libc-test` rather than the mirror `git://repo.or.cz/libc-test` ... we should refer to one or the other consistently just to avoid confusion for folks who may not understand the relationship between the two.

Reply:
Done